Cargo.toml

```diff
@@ -19,12 +18,13 @@ clap-verbosity-flag = { version = "3.0.3", features = ["tracing"] }
 clap_complete = "4.5.55"
 codespan-reporting = "0.12.0"
 colored = "3.0.0"
-crankshaft-docker = "0.2.0"
+crankshaft = { version = "0.4.0", git = "https://github.com/stjude-rust-labs/crankshaft", rev = "b99c2fd" }
+crankshaft-docker = { version = "0.2.0", git = "https://github.com/stjude-rust-labs/crankshaft", rev = "b99c2fd" }
```

Comment:
I'm not sure why this isn't the case, but I would expect this to be exposed under `crankshaft` as the `docker` feature. Is there a reason I'm forgetting why we didn't do it that way?